examples/metrics/kafka-connect-metrics.yaml

```diff
@@ -91,15 +91,25 @@ data:
       help: "Kafka $1 JMX metric type $2"
       type: GAUGE
 
+    #kafka.connect:type=connector-metrics,connector="{connector}"
+    - pattern: 'kafka.(.+)<type=connector-metrics, connector=(.+)><>(connector-class|connector-type|connector-version|status): (.+)'
+      name: kafka_connect_connector_$3
+      value: 1
+      labels:
+        connector: "$2"
+        $3: "$4"
+      help: "Kafka Connect $3 JMX metric type connector"
+      type: GAUGE
+      
     #kafka.connect:type=connector-task-metrics,connector="{connector}",task="{task}<> status"
     - pattern: 'kafka.connect<type=connector-task-metrics, connector=(.+), task=(.+)><>status: ([a-z-]+)'
-      name: kafka_connect_connector_status
+      name: kafka_connect_connector_task_status
```

Comment:
So, from the diff it looks like the `kafka_connect_connector_task_status` replaces the `kafka_connect_connector_status`. Doesn't that mean that the `kafka_connect_connector_status` will be missing now?

Reply:
Sorry, I read the description once again ... so are you saying that there is basically no `kafka_connect_connector_status` ... it needs to be replced by `kafka_connect_connector_task_status` with the right task label?

In that case, do we need to also update the Grafana chart? Or is it not used there?

Reply:
I do not see this metric to be used in the Grafana dashboard examples.

Reply:
Hey sorry for the delay, the task status metrics, is unused by the current Grafana dashboard, I have updated the Grafana dashboard template and moved the changes to the packaging folder as it's required by build.
https://github.com/strimzi/strimzi-kafka-operator/pull/5583